clients/lighthouse-bn/lighthouse_bn.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+if test -f "/hive/input/genesis.ssz"; then
+    if [ -z "$HIVE_ETH2_ETH1_RPC_ADDRS" ]; then
+      echo "genesis.ssz file is missing, and no Eth1 RPC addr was provided for building genesis from scratch."
+      # TODO: alternative to start from weak-subjectivity-state
+      exit 1
+    fi
+fi
+
+mkdir -p /data/testnet_setup
+
+cp /hive/input/genesis.ssz /data/testnet_setup/genesis.ssz
+
+# empty bootnodes file, required for custom testnet setup, use CLI arg instead to configure it.
+echo "[]" > /data/testnet_setup/boot_enr.yaml
+
+echo "${DEPOSIT_CONTRACT_ADDRESS:-'0x1111111111111111111111111111111111111111'}" > /data/testnet_setup/deposit_contract.txt
+echo "${HIVE_ETH2_DEPOSIT_DEPLOY_BLOCK_NUMBER:-'0'}" > /data/testnet_setup/deploy_block.txt
```

Comment:
I had problem running lighthouse because of this line. If `HIVE_ETH2_DEPOSIT_DEPLOY_BLOCK_NUMBER` is not defined,  bash will echo `'0'` to `deploy_block.txt` instead of just `0`. `'0'` is a string, lighthouse expects u64 number and fails to start. 
Solved as expected when `''` is removed.